storybook/src/example-pages/Patronen/beheer-basis.content.ts

```diff
@@ -2,7 +2,8 @@ import { Breadcrumbs, DefinitionList, Header } from "dso-toolkit";
+  label: "",
```

Comment:
Er zou ook wat voor te zeggen zijn om geen property `label` te zetten. We verwachten immers ook dat afnemers `<dso-logo>` doen en geen `<dso-logo label="">`.

Reply:
Goed punt! Ik haal het weg